react/Minicart.tsx

```diff
@@ -58,9 +58,20 @@ const Minicart: FC<MinicartProps> = ({
 }) => {
   const { handles, withModifiers } = useCssHandles(CSS_HANDLES, { classes })
 
-  const { variation } = useMinicartState()
+  const { orderForm }: OrderFormContext = useOrderForm()
+
+  const { push } = usePixel()
+
+  const { variation, open } = useMinicartState()
   const { url: checkoutUrl } = useCheckoutURL()
 
+  useEffect(() => {
+    open && push({
+      event: 'viewCart',
+      items: orderForm.items,
+    })
+  }, [open])
```

Comment:
Running `yarn lint` at the root the linter suggests the following:
```suggestion
  useEffect(() => {
    open &&
      push({
        event: 'viewCart',
        items: orderForm.items,
      })
  }, [open, push, orderForm.items])
```